api-reference/_hidden/GridBaseColumn/isBand.md

```diff
@@ -5,54 +5,16 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies whether the column bands other columns or not.
+Specifies whether the column bands other columns.
```

Comment:
What do you mean when you say that a "column bands other columns"? Are you talking about banded columns (every other column is shaded)?

Reply:
@albertov05 
![image](https://user-images.githubusercontent.com/66619750/91981384-70362b00-ed31-11ea-97ef-969aa03f87b4.png)
Yes, I am talking about banded columns. In this picture, the main band column is `Nominal GDP` and the banded columns are `Total, mln` and `By Sector`. The `By Sector` column is in its turn a band column. Its banded columns are `Agriculture`, `Industry`, and `Services`.

Reply:
Thank you. I understand now. Can we say:

```suggestion
Specifies whether the column organizes other columns into bands.
```

Reply:
Yes.

Reply:
Thanks, please apply my suggestion.